home/.bin/create-feature

```diff
@@ -0,0 +1,54 @@
+#!/usr/bin/env ruby
+require_relative 'feature'
+
+help_text = <<-eoht
```

Comment:
right. for now we just have the one flag, and I don't want to add more unless we want to get real serious